src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Embedded.cs

```diff
@@ -113,6 +178,8 @@ public static void Main()
 }";
 
             CreateCompilation(code, references: new[] { reference }, assemblyName: "Source").VerifyDiagnostics(
+                // error CS0101: The namespace 'Microsoft.CodeAnalysis' already contains a definition for 'EmbeddedAttribute'
```

Comment:
>// error CS0101: The namespace 'Microsoft.CodeAnalysis' already contains a definition for 'EmbeddedAttribute' [](start = 16, length = 109)

The breaking change description says: 
"
1. Previously, we allowed to refer to a `Microsoft.CodeAnalysis.EmbeddedAttribute` type declared in an added module.
    In Visual Studio 2019, this produces a collision with the injected declaration of that type.
"

However, it doesn't look like this scenario refers to the type, yet it is still an error. Consider adjusting description of the breaking change. It isn't really about referring, it is about having this type declared in an added module.